views/apartments/edit.erb

```diff
@@ -0,0 +1,29 @@
+<form method="post" action="/apartment/<%= @apartment.id %>">
+  <input type="hidden" name="_method" value="put">
+  <fieldset>
```

Comment:
Challenge: could you make a loop for this so you don't have to explicitly write out each fieldset/input?